lib/oelite/cookbook.py

```diff
@@ -33,7 +33,7 @@ def __init__(self, baker):
         if not self.db:
             raise Exception("could not create in-memory sqlite db")
         self.db.text_factory = str
-        self.dbc = self.db.cursor()
+        self.dbc = CursorWrapper(self.db.cursor(), profile=False)
```

Comment:
yeah. I plan on introducing __profile,  __profile_sql, __profile_foo variables one can set in local.conf to control whether and which profiling data gets gathered, but just haven't got around to it yet. But being able to turn this on by changing just a single line is still a lot easier than bolting on the rest of the infrastructure.